README.md

```diff
@@ -36,7 +36,7 @@ using ParserCombinator
 
 # the AST nodes we will construct, with evaluation via calc()
 
-abstract Node
+abstract type Node node
```

Comment:
??

Reply:
Oh that looks like `sed` got away from me.